docs/source/contrib_docs/doc-workflow.rst

```diff
@@ -0,0 +1,36 @@
+Understanding our workflow
+==========================
+
+**High level documentation workflow**
+
+1. Identify a documentation change.
+
+   - *Typos:* please go ahead and fix it (or report as a bug).
+   - *Open issues:* leave a note in the issue comments that you
+     are working on the issue.
+   - *New documentation:* open an issue with your idea or suggestion. We'll
+     review the issue and work with you to identify next steps.
+
+2. Update the source file.
+
+3. Commit the change.
+
+4. Test changes locally.
+
+5. Open a pull request.
+
+6. Check response of automated tests.
```

Comment:
Automated tests are rarely relevant to docs.

Do we really need a completely separate workflow for docs changes? Most of the steps are the same as for other kinds of change. Is it feasible to write a single workflow with some details for different scenarios?